friends/src/App.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+import './App.css';
+
+class App extends Component {
+  constructor() {
+    super();
+    this.state= {
+      friends: []
+    };
+  }
+  componentDidMount() {
+    axios
+      .get('http://localhost:5000/friends')
+      .then(response => {
+        this.setState({friends: response.data});
+      })
+      .catch(err => console.log(err));
+  }
+  render() {
+    return (
+      <div className="App">
+       <h1>All My Lambda Friends</h1>
+        {this.state.friends.map(friend => (
+        <div>
```

Comment:
That said, good job getting this to work! Don't forget about adding a `key` prop when you map over something like this in a React app. Thankfully, the friends data comes with a property called `id`, which you could have used.